### DISCUSSION TITLE
quarto serve multiple page notebooks created with the website template

### DISCUSSION THREAD

**Discussion by arnaud-feldmann** [quarto-dev/quarto-cli#701] [Q&A] [answered]
Hello,

Thanks for your marvelous work. It's a very nice evolution of RMarkdown. There's a functionality, through, I'd love to see happen : it's quarto serve on folders with multiple notebooks, in order to allow very basic websites but with interactivity. Is there something like that intended, that would mix the power of `quarto create-project site --type website` and `quarto serve` in order to bring easy dashboards with multiple pages ?

Thanks !! :D

**Comment by dragonstyle** [comment, marked as answer]
You might be able to accomplish what you'd like using our listings feature. For example, this attached sample will enumerate the notebooks from a folder into a table on the home page.

[simple-nb.zip](https://github.com/quarto-dev/quarto-cli/files/8523758/simple-nb.zip)

More about listings here:

https://quarto.org/docs/websites/website-listings.html

**Reply by jjallaire** [reply]
There isn't currently any way to host multiple `quarto serve` targets in a single website (although we will probably create something like this facilitated by iframes at some point).

**Reply by arnaud-feldmann** [reply]
Thanks for you answer ; and keep up the good work !
